### ISSUE TITLE
Moved CLI commands to consolidate packages under moniker templates

### ISSUE THREAD

**Issue by jsteenb2** [influxdata/docs-v2#1087]
These changes are to remove the mention of packages from our public CLI interface. The change incorporates many conventions from popular CLIs in infrastructure (kubectl for instance).

The old command still work, but they are hidden in the CLI. It'd benefit us to remove them from our docs imo, but that is a decision for you all. I'd like to axe those deprecated commands within the next ~month.

##### Relevant URLs
- basically all the pkg docs 😬 
- PR driving changes: https://github.com/influxdata/influxdb/pull/18392

**Comment by sanderson** [comment]
Related to #1088.
